cmd/config/metrics.yml

```diff
@@ -187,3 +187,14 @@
 
 - query: rate ( node_vmstat_pgmajfault[1m] )
   metricName: nodeMajorFaults
+
+# OVS 5m CPU usage average
+- query: avg by (instance) (rate(container_cpu_usage_seconds_total{id=~"/system.slice/ovs-vswitchd.service"}[5m])) * 100
+  metricName: ovsCPUUsagePercent
+  instant: true
+  captureStart: true
```

Comment:
The avg aggregator is not necessary, it's not actually averaging anything as these metrics are broken down by instance. Also, we normally use 2 minutes for rate in order to have a better granularity. This can be simplified in

```
irate(container_cpu_usage_seconds_total{id=~"/system.slice/ovs-vswitchd.service"}[2m])
```

And last, instant and captureStart shouldn't be enabled here, the idea is to index the full timeseries of these metrics, once we've got them indexed, we can use Orion to average the value and detect potential regressions on CPU usage.

Reply:
what do you think of this approach @afcollins ?